packages/api/tests/endpoints/test-endpoints-granules.js

```diff
@@ -101,6 +101,17 @@ test.serial('default returns list of granules', async (t) => {
   });
 });
 
+test.serial(`When accessing any API endpoint with no session information, it returns an HTTP 401 status code
+           and no system information`, async (t) => {
+  const event = { httpMethod: 'GET', headers: {} };
+
+  const response = await granuleEndpoint(event);
+  const expectedResponse = { message: 'Authorization header missing' };
+
+  t.is(response.statusCode, 401);
+  t.deepEqual(JSON.parse(response.body), expectedResponse);
```

Comment:
I would probably say

```
const responseBody = JSON.parse(response.body);
t.is(responseBody.message, 'Authorization header missing');
```

That seems to read a little more clearly, at least to me.